File: sandbox/spring-demo/claims-service/src/main/java/com/maplesure/claims/Claim.java
package com.maplesure.claims;

import java.math.BigDecimal;
import java.time.Instant;

public record Claim(
        long id,
        String policyNumber,
        String holderName,
        BigDecimal amount,
        String description,
        String status,
        Instant submittedAt) {
}

File: repos/claimsportal/claims_service/claim.py
from pydantic import BaseModel


class Claim(BaseModel):
    """A benefit claim filed by a plan member against a group contract."""

    id: int
    policyNumber: str
    holderName: str  # the plan sponsor on the contract the claim is filed against
    memberId: str
    serviceType: str
    amount: float
    description: str
    status: str
    submittedAt: str
